app/helpers/announcements_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module AnnouncementsHelper
+  def announcement_exists?(announcement)
```

Comment:
this method is no longer needed

Reply:
resolved

Reply:
def exists? method can also be removed as it was for announcement exist method  to check existance.... removing it too...